admin-dev/themes/new-theme/js/app/pages/stock/components/overview/products-actions.vue

```diff
@@ -104,6 +95,15 @@
       },
     },
     methods: {
+      isIndeterminate() {
```

Comment:
Because a computed instruction should not transform a property or a data, it's dedicated to render. https://vuejs.org/v2/guide/computed.html#Computed-Properties
It's a bad practice and not allowed in eslint-vue